### ISSUE TITLE
AWS example permissions incorrect

### ISSUE THREAD

**Issue by prydie** [restic/restic#1593]
When following through the AWS example the permissions detailed in the '[Creating a User](http://restic.readthedocs.io/en/latest/080_examples.html#creating-a-user)' I got the following error:

```
root@foo~# restic version
restic 0.8.1
compiled with go1.9.2 on linux/amd64
root@foo:~# restic init
create backend at s3:s3.amazonaws.com/foo-backups/restic failed: client.BucketExists: Access Denied.
```

Looking at the code it seems `s3.Create()` calls `be.client.BucketExists(cfg.Bucket)` ([here](https://github.com/restic/restic/blob/1c9fca6d14d915475cb5d796aac813b30092947e/internal/backend/s3/s3.go#L106)) which requires `s3:GetBucketLocation`.

Modifying the policy to match the following allowed me to successfully run `restic init` and create an initial backup:

```json
{
    "Version": "2012-10-17",
    "Statement": [
        {
            "Sid": "VisualEditor0",
            "Effect": "Allow",
            "Action": [
                "s3:PutObject",
                "s3:GetObject",
                "s3:DeleteObject"
            ],
            "Resource": "arn:aws:s3:::foo-backups/*"
        },
        {
            "Sid": "VisualEditor1",
            "Effect": "Allow",
            "Action": [
                "s3:ListBucket",
                "s3:GetBucketLocation"
            ],
            "Resource": "arn:aws:s3:::foo-backups"
        }
    ]
}
```

**Comment by fd0** [comment]
Thanks for the report!

Ping @Phaeilo, would you mind having a look? Thanks!

**Comment by Phaeilo** [comment]
I can confirm this behaviour. I'm updating the AWS tutorial to include `s3:GetBucketLocation`.

**Comment by fd0** [comment]
Thanks for checking!

**Comment by fd0** [comment]
Resolved in #1598, thanks @Phaeilo!
